ted_sws/notice_transformer/entrypoints/cmd_mapping_suite_transformer.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/python3
+
+import os
+import sys
+
+module_path = os.path.dirname(os.path.realpath(__file__)) + '/../../../'
+sys.path.append(module_path)
+
+import click
+from pathlib import Path
+
+from ted_sws import config
+from ted_sws.data_manager.adapters.mapping_suite_repository import MappingSuiteRepositoryInFileSystem, \
+    METADATA_FILE_NAME
+from ted_sws.notice_transformer.adapters.rml_mapper import RMLMapper
+import ted_sws.notice_transformer.services.notice_transformer as notice_transformer
+
+DEFAULT_MAPPINGS_PATH = 'mappings'
+DEFAULT_OUTPUT_PATH = 'output'
+
+"""
+SETUP:
+1) Clone ted-sws repository on the same directory level as ted-sws-artefacts project.
+2) Go to ted-sws directory.
+3) Run the following commands (it will install the requirements and set the path to rplmapper.jar):
+   # make install && make local-dotenv-file 
+   (.env file must already be created)
+4) Go to ted-sws-artefacts directory.
+5) Create a symlink in ted-sws-artefacts project's root directory:
```

Comment:
Certainly, the symlink is not the right way to do this. 
Rather, this shall be facilitated through `pip install` of the `ted-sws` repository configured through `setup.py`

Reply:
Already answered above: this was implemented to get faster to semantic team (It will be updated so it can be used as Python package)